src/main/groovy/life/qbic/datamodel/datasets/datastructure/folders/nanopore/Fast5SkipFolder.groovy

```diff
@@ -0,0 +1,38 @@
+package life.qbic.datamodel.datasets.datastructure.folders.nanopore
+
+import life.qbic.datamodel.datasets.datastructure.files.nanopore.Fast5File
+import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
+
+/**
+ * A special case of a DataFolder, its name is always fast5_skip.
+ *
+ * Its children field contains either a list of type List<Fast5Files>
```

Comment:
same problem as with pod5 skip

Reply:
Good eyes 👁️ (See my answer to the pod5 skip)